### DISCUSSION TITLE
Why were the patch versions for CVE-2022-27652 and CVE-2022-0532 released so late?

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [cri-o/cri-o#7482] [Q&A]
We are a research team dedicated to Golang, have discovered that CVE-2022-27652 and CVE-2022-0532 were addressed in commit 32d68280019f56fbc3039fda81d719118a991a93 and 9ad033bc06aa8a9c53c0bf9409c58965e6049b93. However, upon analyzing the commit, we observed that the patch versions (v1.24.0) was released after a lapse of over one month. We are interested in understanding the reasons behind this delay in releasing the patch version, as it could potentially impede the prompt dissemination of patches to downstream users. We seek clarification on whether the delay might be attributed to:

1. Issues with testing and CI checking.
2. Other commits requiring inclusion in a single release.
3. By convention, infrequent release of versions.
4. Other reasons.
   We appreciate your attention to this matter and eagerly await your response. Thank you.

**Comment by haircommander** [comment]
this has part to do with CRI-O release cadance. Since we follow the kubernetes project, we don't release minor versions (1.y.0) until after kube does. For https://github.com/cri-o/cri-o/pull/5610 it was released in a minor version in 1.24.0. but the patch was brought back to 1.23.1 quickly. Users wouldn't be expected to use 1.24.0 until kube cut it, so it shouldn't have effected anyone. For https://github.com/cri-o/cri-o/security/advisories/GHSA-4hj2-r2pm-3hc6, the risk wasn't deemed worth backporting, so we waited for the minor release
